File: src/main/java/com/ebuozturk/ecommerce/dto/UpdateAddressRequest.java
package com.ebuozturk.ecommerce.dto;

import lombok.AllArgsConstructor;
import lombok.EqualsAndHashCode;
import lombok.Getter;
import lombok.NoArgsConstructor;

@Getter
@EqualsAndHashCode
@NoArgsConstructor
@AllArgsConstructor
public class UpdateAddressRequest {
    private Long id;
    private String addressName;
    private String phoneNumber;
    private String country;
    private String city;
    private String street;
    private String zipCode;
    private Long userId;

}

File: src/main/java/com/ebuozturk/ecommerce/dto/address/UpdateAddressRequest.kt
package com.ebuozturk.ecommerce.dto.address;

data class UpdateAddressRequest @JvmOverloads constructor(
    val addressName: String,
    val phoneNumber: String,
    val country: String,
    val city: String,
    val street: String,
    val zipCode: String,
    val userId: String,
){

}
